resources-generator/src/main/kotlin/dev/icerock/gradle/MultiplatformResourcesPlugin.kt

```diff
@@ -370,3 +420,9 @@ class MultiplatformResourcesPlugin : Plugin<Project> {
         return manifest.attributes.getNamedItem("package").textContent
     }
 }
+
+internal val KotlinCompilation<*>.allKotlinSourceSetsObservable
+    get() = this.allKotlinSourceSets as ObservableSet<KotlinSourceSet>
+
+internal val KotlinCompilation<*>.kotlinSourceSetsObservable
+    get() = this.kotlinSourceSets as ObservableSet<KotlinSourceSet>
```

Comment:
source sets not gradle's live collections?
this cast is safe for us? @sellmair

Reply:
Yes, this cast is safe, we will think about better plugin APIs in the future!